core/include/cib.h

```diff
@@ -89,6 +101,20 @@ static inline int cib_get(cib_t *__restrict cib)
 }
 
 /**
+ * @brief Get the index of the next item in buffer.
+ *
+ * Usafe version, *must* be called if buffer is not empty!
```

Comment:
Also here and below, the wording is a little bit weird ("*must* be called" sounds a little like you have to call it, even if you don't want to). Better would be "Unsafe version, *must not* be called if buffer is empty!"